lib/physicians/resolvers.ts

```diff
@@ -0,0 +1,51 @@
+import { ApolloError } from 'apollo-server-errors'; // Apollo error
+import { Resolver } from '@lib/resolvers'; // Resolver type
+import { DBErrorCode } from '@lib/db/errors'; // Database errors
+import { PhysicianCreateError } from './errors'; // Physician error
+import { Prisma } from '.prisma/client';
```

Comment:
```suggestion
import { PhysicianCreateError } from '@lib/physicians/errors'; // Physician error
import { Prisma } from '@prisma/client';
```

Avoid relative imports

Reply:
good call, missed that